packages/core/src/App/Containers/ResetEmailModal/reset-email-modal.jsx

```diff
@@ -6,54 +6,26 @@ import { Button, Dialog, Text, Input } from '@deriv/components';
 import { validEmail, getErrorMessages } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
-import { SentEmailModal } from '@deriv/account';
-import { WS } from 'Services';
+import { ConfirmEmailModal } from './confirm-email-modal.jsx';
 
 const ResetEmailModal = ({
     disableApp,
+    email,
     enableApp,
     is_loading,
     is_visible,
     verification_code,
     toggleResetEmailModal,
 }) => {
-    const [is_send_email_modal_open, setIsSendEmailModalOpen] = React.useState(false);
-    const [email_request, setEmailRequest] = React.useState(null);
+    const [confirmResetEmailModal, setConfirmResetEmailModal] = React.useState(false);
+    const [get_error, set_get_error] = React.useState(null);
```

Comment:
```suggestion
    const [get_error, setGetError] = React.useState(null);
```
also change the get_error to sth more meaningful